wrapanapi/systems/ec2.py

```diff
@@ -1187,3 +1246,45 @@ def get_registry_data(self):
                     'registry': first_registry['proxyEndpoint']}
         except (IndexError, KeyError):
             raise NotFoundError("couldn't get registry details. please check environment setup")
+
+    def create_network(self, cidr_block='10.0.0.0/16'):
```

Comment:
I would like to see docblocks on these methods, describing the use of their args.

There's some difference that can cause confusion - in `find_networks` the arg is name_or_id, but in `get_network` its just `name` - but it works with id too, because its using find_networks.

This should be laid out in the docblocks, and I think it would be better ot use `name_or_id` arg consistently.

Reply:
Yes, I was already thinking about it...But other entities had get with name and find with name_or_id so I left it same. Changing it so both will be name_or_id.